### ISSUE TITLE
routing problem

### ISSUE THREAD

**Issue by HoHeo** [OrchardCMS/OrchardCore#15353]
### Describe the bug
I have two servers. Both servers are running orchardcore. When the primary server publishes an article and synchronizes it to the secondary server through MySQL, the secondary server's orchardcore will display that the page is not found after opening the article link. Log in to the secondary server orchardcore background to restart. The link can be opened correctly after publishing the article or restarting the iis website instance. This is a very important issue in cluster load balancing websites. If the secondary server works normally after the main server crashes, but the synchronized article links cannot be opened normally, the consequences will still be serious. Please guys solve it, thank you.

**Comment by HoHeo** [comment]
Hello, is there anyone?

**Comment by HoHeo** [comment]
I need you help.

**Comment by ns8482e** [comment]
It depends on how you have setup two servers. Are you using multi-node single deployment that shares the data or single node multiple deployments with their own data store and you use some kind of sync process?

**Comment by HoHeo** [comment]
Yes, use the mysql tool to synchronize the data to the auxiliary server, and compare the two data through the visualization tool. The table structure and data are exactly the same. However, when opening the link to a newly published article in the secondary server's orchardcore, it will show that the page cannot be found. It can only be opened when the secondary server's orchardcore re-publishes the article, or it can be opened after restarting the iis instance.
